data_summaries/Indic-Instruct.json

```diff
@@ -0,0 +1,483 @@
+      "IndicTrans2"
+    ],
+    "Format": [
+      "Zero-shot"
+    ],
+    "Human Annotation": "No",
+    "Derived from Datasets": [
+      "Anthropic-HHH"
+    ],
+    "Creators": [
+      "AI4Bharat",
+      "Indian Institute of Technology Madras",
+      "Indian Institute of Information Technology, Design and Manufacturing, Kancheepuram",
+      "Flipkart",
+      "University Of Surrey",
+      "Agency for Science, Technology and Research",
+      "National Institute of Information and Communication Technology",
+      "IBM",
+      "Microsoft"
+    ],
+    "Licenses": [
+      {
+        "License": "MIT License",
+        "License URL": "https://huggingface.co/datasets/Anthropic/hh-rlhf"
+      }
+    ],
+    "License Notes": "They utilized a variety of existing datasets, adopting diverse licenses in accordance with the licenses of the source datasets.",
+    "License Verified By": "Vipul",
+    "Dataset Filter IDs": [
+      "hh-rlhf"
+    ]
+  },
+  "indic-instruct-lm_sys": {
+    "Unique Dataset Identifier": "indic-instruct-lm_sys",
+    "Dataset Name": "indic-instruct-data-v0.1",
+    "Paper Title": "Airavata: Introducing Hindi Instruction-tuned LLM",
+    "Dataset URL": "https://huggingface.co/datasets/ai4bharat/indic-instruct-data-v0.1",
+    "GitHub URL": "https://github.com/AI4Bharat/IndicInstruct",
+    "Hugging Face URL": "https://huggingface.co/datasets/ai4bharat/indic-instruct-data-v0.1/viewer/lm_sys",
+    "Papers with Code URL": "",
+    "ArXiv URL": "https://arxiv.org/abs/2401.15006",
+    "Semantic Scholar Corpus ID": "267301288",
+    "Collection": "Indic-Instruct",
+    "Collection URL": "https://github.com/AI4Bharat/IndicInstruct",
+    "Languages": [
+      "English",
+      "Hindi"
+    ],
+    "Task Categories": [
+      "Instructional Question Answering",
+      "Closed-Book Question Answering",
+      "Open-Ended Question Answering",
+      "Natural Language Classification",
+      "Open-Domain Question Answering",
+      "Brainstorming",
+      "Information Extraction",
+      "Summarization",
+      "Creative Writing",
+      "Instruction Following",
+      "Translation",
+      "Information Extraction"
+    ],
+    "Text Sources": [],
+    "Model Generated": [
+      "IndicTrans2"
+    ],
+    "Format": [
+      "Zero-shot"
+    ],
+    "Human Annotation": "No",
+    "Derived from Datasets": [
+      "LMSYS-Chat-1M"
+    ],
+    "Creators": [
+      "AI4Bharat",
+      "Indian Institute of Technology Madras",
+      "Indian Institute of Information Technology, Design and Manufacturing, Kancheepuram",
+      "Flipkart",
+      "University Of Surrey",
+      "Agency for Science, Technology and Research",
+      "National Institute of Information and Communication Technology",
+      "IBM",
+      "Microsoft"
+    ],
+    "Licenses": [
+      {
+        "License": "LMSYS-Chat-1M",
+        "License URL": "https://huggingface.co/datasets/lmsys/lmsys-chat-1m#lmsys-chat-1m-dataset-license-agreement"
```

Comment:
For Robert: how should we interpret this? They have a fairly open license here, but at the same time the dataset includes model responses from OpenAI and other models with strict terms. Should we separately add those terms to this list?

Reply:
Yep, that license is ok for all purposes, no share alike, no attribution.

I think we've had many cases where we have a permissive license + OpenAI - how do we handle those? Let's do the same here.